### DISCUSSION TITLE
Update latest execution result in web GUI

### DISCUSSION THREAD

**Discussion by yulipumal** [jlesage/docker-freefilesync#9] [General]
I followed the [Scheduled Batch Jobs](https://github.com/jlesage/docker-freefilesync#scheduled-batch-jobs) tutorial.

I have managed to have a cron job that starts every hour. But unfortunately, the information of the last execution and the log is not displayed on the web interface.

![grafik](https://github.com/jlesage/docker-freefilesync/assets/122391249/5ca1226d-bcc4-439e-9791-eca7ba0e13ba)
here: 14 days, eventhough it was executed 1 hour ago. The logs in /config/xdg/config/FreeFileSync/Logs on the other hand are correct and show every hour.

And also, the last log linked in the web interface, is not correct:
![grafik](https://github.com/jlesage/docker-freefilesync/assets/122391249/41eadc1f-1e79-43d0-b091-567e4e950875)

**Comment by jlesage** [comment]
This seems to be expected.  Scheduled batch jobs and the main UI are not connected together.  It's currently 2 different ways/interfaces to perform a synchronization.

A "standard" installation of FreeFileSync in Windows/Linux/Mac would have the same behaviour: if you have the application window open while a scheduled batch job (https://freefilesync.org/manual.php?topic=schedule-batch-jobs) runs, the opened window won't be updated.

I can check if there is a way to force a refresh of the UI, but I'm not the developper of FreeFilySync itself.  You may use the FreeFileSync forum to propose your idea: https://freefilesync.org/forum/
